shapely/ops.py

```diff
@@ -332,14 +329,9 @@ def snap(g1, g2, tolerance):
     tolerance : float
         The snapping tolerance
 
-    Example
-    -------
-    >>> square = Polygon([(1,1), (2, 1), (2, 2), (1, 2), (1, 1)])
-    >>> line = LineString([(0,0), (0.8, 0.8), (1.8, 0.95), (2.6, 0.5)])
-    >>> result = snap(line, square, 0.5)
-    >>> result.wkt
-    'LINESTRING (0 0, 1 1, 2 1, 2.6 0.5)'
+    Refer to `shapely.snap` for full documentation.
```

Comment:
the usual way to cross-reference another function is to do this:
```suggestion
    Refer to :func:`shapely.snap` for full documentation.
```